File: pt-service.php
#!/usr/local/bin/php
<?php
while(1){
    $pid = getmypid();
    $msg = $pid." | ".date("h:i:s")."\n";
    echo $msg;
    file_put_contents("/tmp/pt_service.log",$msg,FILE_APPEND);
    sleep(1);    
}

File: pt-service.py
#!/usr/bin/python
import time
import os
pid = os.getpid()

while 1:    
    f = open("/tmp/pt_service.log","a+")
    f.write(str(pid)+" | "+str(time.time())+"\n")
    f.close()
    time.sleep(1)
